packages/flutter/pubspec.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
   vector_math: 2.1.0
   sky_engine:
     sdk: flutter
+  intl: 0.17.0
```

Comment:
I just came across this https://github.com/flutter/flutter/pull/83304#discussion_r643360023.

Appreciate providing feedback about the way forward.